src/vs/workbench/services/views/browser/viewDescriptorService.ts

```diff
@@ -608,7 +616,7 @@ export class ViewDescriptorService extends Disposable implements IViewDescriptor
 
 		// Execute View Container Movement
 		for (const [container, location] of viewContainersToMove) {
-			this.moveViewContainerToLocation(container, location);
+			this.doMoveViewContainerToLocation(container, location, undefined, true);
```

Comment:
Shall we follow the same pattern while moving views here -

https://github.com/microsoft/vscode/blob/51895f6397ebfa712ed133f9670d7c79e8f95599/src/vs/workbench/services/views/browser/viewDescriptorService.ts#L588

Reply:
If you agree, you can change this for next milestone as debt